src/containers/Review/ReviewHomeWrapper.tsx

```diff
@@ -58,9 +67,59 @@ const ReviewHomeWrapper: React.FC<{
           fullApplicationStructure={fullApplicationStructure}
         />
       )}
-    </>
+    </Container>
+  )
+}
+
+interface ReviewHomeProps {
+  templateCode: string
+  applicationName: string
+  orgName?: string
+}
+
+const ReviewHomeHeader: React.FC<ReviewHomeProps> = ({
+<<<<<<< HEAD
+  templateCode,
+  applicationName,
+  orgName,
+}) => {
+  const { push } = useRouter()
+  return (
+    <div id="review-home-header">
+      <Label
+        className="simple-label clickable"
+        onClick={() => push(`/applications?type=${templateCode}`)}
+        icon={<Icon name="chevron left" className="dark-grey" />}
+      />
+      <Header
+        as="h2"
+        content={applicationName}
+        subheader={<Header as="h5" content={orgName || strings.TITLE_NO_ORGANISATION} />}
+      />
+    </div>
```

Comment:
Oops - merge conflict markers still in here.

Reply:
Not sure how did this happen. I had fixed the exact same conflicts yesterday.